use-cases/using-SRIOV-and-kata.md

```diff
@@ -41,6 +41,9 @@ In order to setup your host for SR-IOV, the following has to be true:
   Controller (NIC). The section [Checking your NIC for SR-IOV](#checking-your-nic-for-sr-iov) describes how to assess if you need to make NIC changes and how to make
   the necessary changes.
 
+Besides, you need to enable the NIC driver in your guest kernel config:
+eg. mlx5 for menallox NIC
+
```

Comment:
I'm not really sure I understand this comment. Also, should that be `Mellanox*` ?

/cc @amshinde, who wrote this document.